hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -1684,4 +1684,8 @@ public static void destroyAllDistributedObjects(HazelcastInstance hz) {
             object.destroy();
         }
     }
+
+    public static boolean isSolaris() {
```

Comment:
Maybe the `TestEnvironment` class could be a better destination for this method.